### PR TITLE
feat: add `sidekiq` cli autocompletion specs 🚀 

### PR DESCRIPTION
**Pull request by omarluq**
Hello everyone 😄 
I'm working on adding an autocompletion spec for [sidekiq](https://github.com/sidekiq)!

### CODE REVIEW

**Comment by omarluq** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by omarluq** [comment]
Hey @fedeci @mschrage! if you guys are okay with this PR I would like to make a series of PRs adding some more missing rails cli commands that would be nice to have! thinking of adding [ViewComponents](https://github.com/ViewComponent/view_component)  generators next 🚀

File: src/sidekiq.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const completionSpec: Fig.Spec = {
```
Comment (fedeci) [inline src/sidekiq.ts:1]:
```suggestion
import { filepaths } from "@fig/autocomplete-generators";

const completionSpec: Fig.Spec = {
```

File: src/sidekiq.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+      name: ["--queue", "-q"],
+      description: "Queues to process with optional weights",
+      args: [{ name: "QUEUE" }, { name: "WEIGHT", isOptional: true }],
+    },
+    {
+      name: ["--require", "-r"],
+      description: "Location of Rails application with jobs or file to require",
+      args: { name: "PATH|DIR" },
```
Comment (fedeci) [inline src/sidekiq.ts:28]:
```suggestion
      args: { name: "PATH|DIR", template: ["folders", "filepaths"] },
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks @omarluq! It would be great to get some rails completions, I'll be happy to review those too!

**Comment by omarluq** [comment]
@fedeci Awesome, I implemented the changes you recommended! 
I also will have another PR up in a few days hopefully 😄

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks!

**Pull request merged**